src/mpid/ch4/netmod/ofi/ofi_events.c

```diff
@@ -381,13 +381,14 @@ int MPIDI_OFI_get_huge_event(struct fi_cq_tagged_entry *wc, MPIR_Request * req)
     }
 
     int nic = 0;
-    int vni_src = recv_elem->remote_info.vni_src;
-    int vni_dst = recv_elem->remote_info.vni_dst;
+    /* This is receiver, so dst is local and src is remote */
+    int vni_local = recv_elem->remote_info.vni_dst;
+    int vni_remote = recv_elem->remote_info.vni_src;
```

Comment:
Is it necessary to introduce a new pair of names? All you need is a comment that this function is called on `dst`, right?

Reply:
I don't think the name changes helped.